pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -5,9 +5,14 @@ if test -f /proc/vmcore; then
     exit 0;
 fi
 
-default_cgroup_memory_limit=838860800 #800M
+virttype=`cat /run/eve-virt-type`
 default_cgroup_cpus_limit=1
 
+if [ $virttype = "kubevirt" ]; then
```

Comment:
This is one of 2 (more?) `if` special statements for `kubevirt`. Would it be better to capture them all into one place, make it a `case` statement for all known HV?

e.g.

```bash
case $virttype in
kubevirt)
  # set lots of things
kvm|xen)
  # set other things
*)
  # error
esac
```

Would that make this any clearer?

Reply:
We could do that. But this is just a temporary fix and need to be cleaned up anyway.

Reply:
Fair enough. Even so, nothing is as permanent as something temporary. 😬